description_templates/postgres:vshn.gotmpl

```diff
@@ -0,0 +1,3 @@
+VSHN-managed PostgreSQL
```

Comment:
It's actually called "PostgreSQL by VSHN", see https://products.docs.vshn.ch/products/appcat/postgresql.html

```suggestion
PostgreSQL by VSHN
```